src/LinuxTracing/UprobesUnwindingVisitorDynamicInstrumentationTest.cpp

```diff
@@ -248,6 +271,24 @@ TEST_F(UprobesUnwindingVisitorDynamicInstrumentationTest,
     EXPECT_THAT(actual_function_call.registers(), ElementsAre());
   }
 
+  {
+    UserSpaceFunctionExitPerfEvent function_exit4{
+        .timestamp = 800,
+        .data =
+            {
+                .pid = kPidUnkown,
+                .tid = kTidUnkown,
+            },
+    };
+
+    EXPECT_CALL(return_address_manager_, ProcessFunctionExit(0)).Times(0);
+    orbit_grpc_protos::FunctionCall actual_function_call;
+    EXPECT_CALL(listener_, OnFunctionCall).Times(0);
+    PerfEvent{function_exit4}.Accept(&visitor_);
+    Mock::VerifyAndClearExpectations(&return_address_manager_);
+    Mock::VerifyAndClearExpectations(&listener_);
+  }
+
   {
```

Comment:
I think there should be an additional unit test covering the clone exit / new task cases.